### DISCUSSION TITLE
This button doesn't always send the "Magic Link" template

### DISCUSSION THREAD

**Discussion by askkaz** [supabase/supabase#27425] [Feature Requests]
We have walked a lot of Supabase users through setting up their auth emails and it is frequently a source of confusion that this button:
<img width="288" alt="Screenshot 2024-06-20 at 12 44 40 PM" src="https://github.com/supabase/supabase/assets/7975721/3dd4608b-d299-444e-8600-2ba3b325a837">
sends the "Confirm signup" template if the user has not yet confirmed their email and not the "Magic Link" template.
<img width="840" alt="Screenshot 2024-06-20 at 12 45 08 PM" src="https://github.com/supabase/supabase/assets/7975721/c7323d29-7f55-4685-99d6-92a6a70d3f2b">

I think it would help users getting started if the initial menu was context aware enough to display the correct option ("confirm signup").

bonus nit - template names should match, see "password recovery" vs. "reset password"

double bonus nit - email template tab names should all match casing ("Confirm signup" vs "Magic Link")

**Comment by encima** [comment]
It looks like an Issue was created from this so work will be done to implement this feedback, thanks!

We will close this and track progress in the issue here: #27475
